content/guides/kubernetes/velero-gs.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Getting Started with Velero
+linkTitle: Velero
+weight: 3
+topics:
+- Kubernetes
+tags:
+- Velero
+patterns:
+- Deployment
+team: 
+- Tiffany Jernigan
+---
+
+What do you do if you lose state in your cluster, or something went very wrong and you need to revert it or move my resources to another cluster? Are you out of luck?
+
+That’s where Velero comes in. Velero is an open source tool to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes. This guide will show you how to deploy Velero to your Kubernetes cluster, create backups, and recover from a backup after something goes wrong in the cluster.
+## Prerequisites
+Before you get started you will need to do the following:
+* **Create a Kubernetes cluster**: You need a Kubernetes cluster, v1.10 or later, with DNS and container networking enabled. This guide uses a Google Kubernetes Engine (GKE) Linux cluster with Kubernetes version 1.16.5 and Velero 1.5.3. As of the writing of this guide, Velero [does not officially support Windows](https://velero.io/docs/v1.5/basic-install/#velero-on-windows).
+* **Install [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) on our local machine**: You probably already have that one if you're working with Kubernetes.installed locally
+**Verify the cluster has a [storage class**](https://kubernetes.io/docs/concepts/storage/storage-classes/): Check if you have a storage class with `kubectl get storageclasses`. You need one to create persistent volumes.
```

Comment:
Looks like you're missing a leading bullet point here - GitHub rendered L22 inline with L21.

Reply:
Thank you!